pkg/config/config.go

```diff
@@ -395,13 +399,13 @@ func (c *Config) GetRepoOutputDir(env string) (string, error) {
 	return path.Join(base, c.RepoState.Metadata.OutputDir, env), nil
 }
 
-// GetRepoApplicationBaseDir returns the directory where application
+// GetRepoApplicatiosOutputDir returns the directory where application
 // resources are stored
-func (c *Config) GetRepoApplicationBaseDir() (string, error) {
+func (c *Config) GetRepoApplicatiosOutputDir() (string, error) {
```

Comment:
Typo, should be: `GetRepoApplicationsOutputdir`, unless you are trying to form your spanish soul rock band `Applicatios`, in which case, carry on.

Reply:
Now that you mention it..

Reply:
We werent going to announce it yet, but surprise! Me and @yngvark are the Los Applicatios! 🎉🎉🎉🎉

Reply:
I am already a fan 🎧 ❤️